### ISSUE TITLE
header of downloadResult

### ISSUE THREAD

**Issue by jebai0521** [itinance/react-native-fs#877]
in some case, we need the download task's response header, such as  ETag, Modify-Since

we will use the headers for next download

**Comment by Hirurgo** [comment]
Yep, in my case i also need to get file name from Content-Disposition header
